src/generate/renders/tests/CliTestScenario.ts

```diff
@@ -152,7 +154,7 @@ export class CliTestScenario extends TemplateBase {
                                         !disabled &&
                                         check.indexOf('json.loads') >= 0
                                     ) {
-                                        header.addImport('json');
+                                        template.header.addImport('json');
```

Comment:
I don't see where the template.header is defined.

Reply:
it's a variable  member of class CliTestScenario